include/sampling/ellipsoid.hpp

```diff
@@ -0,0 +1,39 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 2012-2020 Vissarion Fisikopoulos
+// Copyright (c) 2018-2020 Apostolos Chalkis
+// Copyright (c) 2021- Vaibhav Thakkar
+
+//Contributed and/or modified by Vaibhav Thakkar, as part of Google Summer of Code 2021 program.
+
+
+#ifndef SAMPLERS_ELLIPSOID_HPP
+#define SAMPLERS_ELLIPSOID_HPP
+
+#include "sphere.hpp"
+#include "Eigen/Eigen"
+
+
+template <typename Point>
+struct GetPointInDellipsoid
+{
+    template <typename NT, typename VT, typename MT, typename RandomNumberGenerator>
+    inline static Point apply(unsigned int const& dim,
+                              VT const& eigenvals_inv_sqrt, // sqrt of inverse of eigenvals of matrix A in (x'Ax <= 1)
+                              MT const& EigenVecs,          // eigenvecs of matrix A in (x'Ax <= 1)
+                              RandomNumberGenerator &rng)
```

Comment:
`RandomNumberGenerator& rng` to follow the rest of the style

Reply:
done 👍🏽